Grant Application: W3b-Stitch

rocketjays-cmyk opened the pull request:
#### Project Abstract

W3b Stitch is a decentralized trust engine designed to provide verifiable authenticity for media, credentials, and identity across digital platforms. By combining blockchain anchoring, verifiable credentials (VCs), and decentralized identifiers (DIDs), the project creates a scalable trust layer for the internet. W3b Stitch enables selective disclosure, privacy-preserving verification, and dynamic trust recalculation, helping organizations and individuals prove authenticity without relying on centralized gatekeepers.

The system is built around modular components: media verification, credential provenance, decentralized identity, and cross-ledger anchoring. Its applications range from journalism and content platforms to compliance, enterprise credentialing, and digital communities. By making verification seamless and interoperable with existing web standards, W3b Stitch reduces misinformation, strengthens reputations, and enables a more trustworthy digital ecosystem. This work was filed with USPTO since July 2025. Any related work that was conceived after that date may lack the robust legal protection we have.


#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (Polkadot AssetHub (USDC & DOT) address in the application and bank details via email, if applicable).
- [x] I understand that an agreed upon percentage of each milestone will be paid in vested DOT, to the Polkadot address listed in the application.
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

rocketjays-cmyk commented:
I have read and hereby sign the Contributor License Agreement.

Noc2 reviewed (changes requested):
Thanks a lot for the application and for your interest in our grants program. However, the formatting of your application does not match our template. Could you please take a look at it and update the application accordingly? https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md Also, teams need to take at least 50% in vested DOTs.

keeganquigley reviewed (changes requested):
Hi @rocketjays-cmyk the application still contains a lot of boilerplate template text. Can you remove the deliverables in the milestone table and replace with your own? Same goes with milestone 2.

rocketjays-cmyk commented:
Updated even further for better clarification. Added another commit better organizing milestones and added a tech stack graph. Hopefully these updates make it more uniform/compliant.

rocketjays-cmyk commented:
I have created a new repo called W3b-Stitch-Dot. Will be used for the MVP of the project so it can be streamlined. Adding details and code to it actively.

diogo-w3f commented:
@rocketjays-cmyk Thanks for the grant application. I have a few quick questions:

* What is your team’s software development experience? Any prior work on GitHub or other repos, or professionally shipped software? How do you plan to structure development and delivery?
* Are you aware of KILT and other DID/VC protocols in the Polkadot ecosystem? Do you plan to integrate with them (issuance, verification, revocation)?
* Polkadot plans to add native EVM compatibility, which could enable EVM IP protocols (e.g., Story and Bernstein) to run on Polkadot. How do you view this—opportunity or competition? Any integration/synergy plan?
* What is your go‑to‑market strategy and business model? How will you monetize and make this sustainable long‑term?

Thanks!

rocketjays-cmyk commented:
W3b stitch now lives at app.w3bstitch.com . The repository has a fully functioning production build.

diogo-w3f commented:
@rocketjays-cmyk please note that the grants guidelines [have recently changed](https://github.com/w3f/Grants-Program/pull/2667)
. It would be helpful if you could provide information showing how your project fits what we’re looking for, and how it avoids the categories listed under What Doesn’t Fit Our Program.

rocketjays-cmyk commented:
Hello guys I will be closing this application. I will be pursuing other opportunities that better fits the kind of project I am working on. I want to thank the web3f team for their advice I’ve learned a lot. I’ve just realized I need to take a different route. @keeganquigley @diogo-w3f @semuelle @Noc2 .
